src/core/Akka.Streams/Dsl/Framing.cs

```diff
@@ -328,15 +328,7 @@ public DelimiterFramingStage(ByteString separatorBytes, int maximumLineBytes, bo
                 _separatorBytes = separatorBytes;
                 _maximumLineBytes = maximumLineBytes;
                 _allowTruncation = allowTruncation;
-
-                Shape = new FlowShape<ByteString, ByteString>(_in, _out);
             }
-            
-            private readonly Inlet<ByteString> _in = new Inlet<ByteString>("DelimiterFraming.in");
-
-            private readonly Outlet<ByteString> _out = new Outlet<ByteString>("DelimiterFraming.in");
-
-            public override FlowShape<ByteString, ByteString> Shape { get; }
```

Comment:
Not sure why I added those properties since they're already provided by the base class...